template/.renovaterc.json.jinja

```diff
@@ -133,10 +133,12 @@
       "datasourceTemplate": "pypi",
       "fileMatch": [
         "^\\.github/workflows/.+\\.yml$",
-        "^\\.renovaterc\\.json$"
+        "^\\.renovaterc\\.json$",
+        "^Makefile$",
```

Comment:
There is a redundant trailing comma here.

Reply:
Got it fixed.